### PR TITLE
Inlining of references

### PR DESCRIPTION
**Pull request by TheDauntless**
Inlining references (see issue #590 )

### CODE REVIEW

File: Document/0x06g-Testing-Network-Communication.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ $ unzip app-name.ipa
 
 `Info.plist` file can be found in the `Payload/BundleName.app/` directory of the extract. It’s a binary encoded file and has to be converted to a human readable format for the analysis. 
 
-`plutil`<sup>[6]</sup> is a tool that’s designed for this purpose. It comes natively with Mac OS 10.2 and above versions.
+`[plutil](https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man1/plutil.1.html "OS X Man Pages - Plutil")` is a tool that’s designed for this purpose. It comes natively with Mac OS 10.2 and above versions.
```
Comment (Sjord) [inline Document/0x06g-Testing-Network-Communication.md]: The backticks are misplaced here. It should be 

```
[`plutil`](...)
```

File: Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md [hunk 2]
```diff
@@ -275,7 +275,7 @@ sys.stdin.read()
 
 -- TODO [Describe how to assess this given either the source code or installer package (APK/IPA/etc.), but without running the app. Tailor this to the general situation (e.g., in some situations, having the decompiled classes is just as good as having the original source, in others it might make a bigger difference). If required, include a subsection about how to test with or without the original sources.] --
 
--- TODO [Confirm purpose of remark "Use the &lt;sup&gt; tag to reference external sources, e.g. Meyer's recipe for tomato soup<sup>[1]</sup>."] --
+-- TODO [Confirm purpose of remark "Use the tag to reference external sources, e.g. Meyer's recipe for tomato soup."] --
```
Comment (Sjord) [inline Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md]: What does this mean? Maybe we can remove it?

File: Document/0x06g-Testing-Network-Communication.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ ATS restrictions can be disabled by configuring exceptions in the Info.plist fil
 * disable PFS or
 * allow connections to local domains
 
-ATS exceptions can be applied globally or per domain basis. The application can globally disable ATS, but opt in for individual domains. The following listing from Apple Developer documentation shows the structure of the `NSAppTransportSecurity` dictionary<sup>[1]</sup>.
+ATS exceptions can be applied globally or per domain basis. The application can globally disable ATS, but opt in for individual domains. The following listing from Apple Developer documentation shows the structure of the `[NSAppTransportSecurity](https://developer.apple.com/library/content/documentation/General/Reference/InfoPlistKeyReference/Articles/CocoaKeys.html#//apple_ref/doc/plist/info/NSAppTransportSecurity "API Reference NSAppTransportSecurity")` dictionary.
```
Comment (Sjord) [inline Document/0x06g-Testing-Network-Communication.md]: The backticks here should be inside the link.

File: Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md (hunk 2 above)
Comment (TheDauntless) [inline Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md]: Those are placeholders I think. I will remove them.

**Pull request merged**